test/remote/gateways/remote_payment_express_test.rb

```diff
@@ -138,6 +138,7 @@ def test_transcript_scrubbing
 
     assert_scrubbed(@credit_card.number, clean_transcript)
     assert_scrubbed(@credit_card.verification_value.to_s, clean_transcript)
+    assert_scrubbed(@gateway.options[:password].to_s, clean_transcript)
```

Comment:
Okay, I'll drop this at this point, but both the fixture and `@gateway.options[:password]` are already strings, so the `.to_s` isn't needed. I know we're trying to patch this quickly, but I'd rather take a few extra moments to make sure the code doesn't suffer than swing back later.